lib/src/interfaces/Options.ts

```diff
@@ -753,4 +753,10 @@ setRoot: {
    * #### (iOS specific)
    */
   rootBackgroundImage?: ImageRequireSource;
+  /**
+   * Enable or disable automatically blurring focused input, dismissing keyboard on unmount
+   * #### (Android specific)
+   * @default false
```

Comment:
Just curious, is there a reason why this was defaulted to false? I ran into the Android issue where the keyboard would stay up after navigation events and was wondering if that is ever expected behavior? It would definitely make my life easier if this was defaulted to true and I didn't have to override it on all of the navigation actions.

Reply:
yeah I defaulted it to false because in the sense of expected behavior its technically a breaking change, so I wanted to have it only be opt in. However if you'd like to have it be the default behavior for your project, I'm fairly sure that you can configure that to be the case [with existing APIs provided](https://wix.github.io/react-native-navigation/#/docs/styling?id=enabling-persistent-styling-properties) (note I didn't actually verify this but it seems logically sound 😄)

Reply:
Ahh interesting. I figured it was a bug because it was Android specific and we were only seeing the keyboard stay open after navigation events when we had a side menu layout. When we changed the layout to be a stack based app, then we started seeing the keyboard being dismissed on navigation events.

Either way, I think the documentation you referenced looks like it should work. Thanks a lot!